### DISCUSSION TITLE
New function from 0.11: "numberItem+": 1

### DISCUSSION THREAD

**Discussion by pgswow** [pocketbase/pocketbase#1883] [Q&A] [answered]
Hello!
Forgive me for distracting from the main work, I wanted to try a new function of the autoincrement.
There is a collection with a field number, I want to add 1 to existing values:
await locals.pb.collection('servers').create({ 'serverID+': 1 });

But I get all new records with value 1
Are I using a new function correctly?

PB 0.12.2, js-sdk 0.11, sveltekit

**Comment by ganigeorgiev** [comment, marked as answer]
The `+` and `-` field data modifiers are not for autoincrement.
They **add** or **subtract** from the existing record value the submitted number.
In your case, because this is a create operation, the existing value of `serveID` is 0, so the end `serveID` would be 1.

**Reply by pgswow** [reply]
I understand you, thanks! =)
